docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -295,26 +363,74 @@ UnusedMethodRule recognizes these entry points.
 * Any method targeted during invocation
 
 #### Definition
+RemoveUnusedMethod uses sources like ApexFlsViolationRule does, but the RemoveUnusedMethod sinks are different. Instead of seeking DML operations that occur in the course of a path, RemoveUnusedMethod sinks track all method invocations that occur on that path and use that information to identify methods that are never invoked.
 
-UnusedMethodRule uses sources like ApexFlsViolationRule does, but the UnusedMethodRule sinks are different. Instead of seeking DML operations that occur in the course of a path, UnusedMethodsRule sinks track all method invocations that occur on that path and use that information to identify methods that are never invoked.
-
-#### Interpreting UnusedMethodRule Results
+#### Interpreting RemoveUnusedMethod Results
 
 Match any violation message that you receive with this case to understand more about the violation.
 
 *Common Case*
 
 > Method %s in class %s isn’t used in any path from any recognized entry point.
  
-Parameter Explanation:
+Explanation:
 
 Because no invocations of the indicated method were found in the paths originating from the identified entry points, the method is unnecessary and can be deleted.
 
-#### UnusedMethodRule Limitations
+#### RemoveUnusedMethod Limitations
 
-- UnusedMethodRule works on static methods and instance methods. Constructors aren't detected.
+- RemoveUnusedMethod works on static methods and instance methods. Constructors aren't detected.
 - Global methods are intentionally excluded because their external usage is assumed.
-- If the set of files included in `--target` is smaller than the files included in `--projectdir`, then `UnusedMethodRule` can return unexpected results. For that reason, we recommend running this rule against your whole codebase, not against individual files.
+- If the set of files included in `--target` is smaller than the files included in `--projectdir`, then `RemoveUnusedMethod` can return unexpected results. For that reason, we recommend running this rule against your whole codebase, not against individual files.
+
+### UseWithSharingOnDatabaseOperation
+
+The UseWithSharingOnDatabaseOperation rule identifies database operations in classes annotated as `without sharing`. It also warns of database operations in classes that inherit with sharing implicitly instead of explicitly using `inherited sharing`. 
+
+With Salesforce sharing rules, you can control who has access to which records, but it is your responsibility to ensure that your Apex code respects sharing rules. To do this, declare classes with a sharing model.
+
+* `with sharing` causes database transactions in a class to respect sharing rules. It is the default.
```

Comment:
"It is the default" is a little misleading. It's more accurate to say it's the default recommendation.

Reply:
@jfeingold35 Added recommendation